src/photos/targets/browser/doctypes.js

```diff
@@ -36,8 +36,7 @@ export default {
     doctype: DOCTYPE_ALBUMS,
     attributes: {
       name: {
-        type: 'string',
-        unique: true
```

Comment:
I had to remove this because the `validate` function of cozy-client is buggy — when we update the album, it will find that one album already has the same name, but fail to realize that it's the same one.

Reply:
Oh! Didn't see this comment before posting mine... 

Do you created an issue?

Reply:
I have now : https://github.com/cozy/cozy-client/issues/335